pkg/receive/handler.go

```diff
@@ -334,21 +340,22 @@ func (h *Handler) receiveHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	tenant := r.Header.Get(h.options.TenantHeader)
-	if tenant == "" {
-		tenant = h.options.DefaultTenantID
-	}
-
-	// TODO(yeya24): handle remote write metadata.
-	// exit early if the request contained no data
+	// Exit early if the request contained no data. We don't support metadata yet. We also cannot fail here, because
+	// this would mean lack of forward compatibility for remote write proto.
 	if len(wreq.Timeseries) == 0 {
-		level.Debug(h.logger).Log("msg", "empty timeseries from client", "tenant", tenant)
+		// TODO(yeya24): Handle remote write metadata.
+		if len(wreq.Metadata) > 0 {
+			// TODO(bwplotka): Do we need this error message?
+			level.Debug(tLogger).Log("msg", "only metadata from client; metadata ingestion not supported; skipping")
+			return
+		}
+		level.Debug(tLogger).Log("msg", "empty remote write request; client bug or newer remote write protocol used?; skipping")
```

Comment:
Does this warrant being logged at a level other than debug? Perhaps info|warn since we would perhaps want to know if this occurs in production where it would likely get buried by the logger?

Or perhaps we wouldn't care, in which case we might as well remove the log entirely?

Reply:
You can ask the same for all debug logs 🙃

Sometimes we care: When we want to check why we have some many request but not many samples, or maybe we want to let tenant know that he is using to new protocol.
Mosty we don't: Sending newer protocol should not page us during night, this is a user error, than can be buried with logs... 

>

Reply:
> You can ask the same for all debug logs

Indeed, this just stood out to me as something that might be useful as an info log but for sure we wouldnt not want to be paged or have a user error spam the logs. Explanation makes sense!